sdk/include/psdk/shellapi.h

```diff
@@ -643,6 +647,7 @@ SHTestTokenMembership(
 #endif
 
 #ifdef UNICODE
+
 #define NOTIFYICONDATA_V1_SIZE NOTIFYICONDATAW_V1_SIZE
 #define NOTIFYICONDATA_V2_SIZE NOTIFYICONDATAW_V2_SIZE
 #define NOTIFYICONDATA_V3_SIZE NOTIFYICONDATAW_V3_SIZE
```

Comment:
Adding conditions here does not seem mandatory, but could be done for consistency.